tmt/steps/report/html/template.html.j2

```diff
@@ -105,8 +157,21 @@
 <div>
 <h1>{{ plan.name }}</h1>
 {% if plan.summary %}<h2>{{ plan.summary }}</h2>{% endif %}
+
 {% if results %}
-<table>
+<table id="filters">
+    <tr>
+        <td><label><input type="checkbox" id="filter_pass" onclick="filter_checkbox(this);" checked>Pass</label></td>
```

Comment:
```suggestion
        <td><label><input type="checkbox" id="filter_pass" onclick="filter_checkbox(this);" checked>pass</label></td>
```

Test results are shown in lowercase, let's keep the checkbox names consistent with them.

Reply:
I personally preferred the uppercase version as it seemed slightly more readable, but I've changed it to lowercase per your suggestion.

Reply:
Thanks.